### DISCUSSION TITLE
Auto import ref in setup typescript

### DISCUSSION THREAD

**Discussion by Abner-Corona** [quasarframework/quasar#11127] [General - Components / Directives / etc] [answered]
Hi, i have a problem imported ref hook in script setup, the sugerences dont show "Add ref to existing import declaration vue"
like the exemple below, thanks for the help!!
![image](https://user-images.githubusercontent.com/51718510/139006941-d3d11abb-406e-494d-9b0f-9d6ae51ce523.png)
![image](https://user-images.githubusercontent.com/51718510/139007070-9d6a60b3-31b3-416a-8000-ee7a78f6a869.png)

**Comment by rfox12** [comment, marked as answer]
`ref` is from `vue`
`import { onMounted, ref } from 'vue';`

This is not a Quasar issue.  This is an issue with your IDE setup.
